lab-carolina/server.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+
+const http = require('http');
+
+
+const Song = require('./model/song.js');
+const Router = require('./lib/router.js');
+const storage = require('./lib/storage.js');
+
+const PORT = process.env.PORT || 3000;
+const router = new Router();
+
+router.get('/api/song', function(req, res) {
```

Comment:
These routes should be within another file (i.e. `song-routes.js`), and you should then pass your router object from line 12 into the require statement for that file: i.e. `require('./lib/song-routes')(router)`.